index.bs

```diff
@@ -396,3 +422,81 @@ Acknowledgements {#ack}
 
 Cure53's [[DOMPURIFY]] is a clear inspiration for the API this document
 describes, as is Internet Explorer's {{window.toStaticHTML()}}.
+
+
+
+Appendix A: Interactions between Sanitizer And Trusted Types {#tt}
+======================================================
+
+<em>This section is not normative.</em>
+
+[[TrustedTypes]] can interact in helpful ways with the Sanitizer API. How to
+best do so will likely depend on the intended use of both. Because the
+Sanitizer API is meant to remove all scripting content, the default is to not
+consider the Sanitizer Trusted Types sensitive.
+
+The predominant use case would be to sanitize strings and use the resulting
+fragments directly, just as without Trusted Types:
+
+```js
+const untrusted_input = "....";
+some_node.AppendChildren(sanitizer.santize(untrusted_input));
+```
+
+For cases where handling of a string is desired, a sanitizing policy can easily
+be created using the `bound` method.
+
+```js
+const my_config = { ... };
+const sanitizing_policy = trustedTypes.createPolicy("apolicyname", {
+  createHTML: new Sanitizer(my_config).bound() });
+
+const untrusted_input = "....";
+some_node.innerHTML = sanitizing_policy.createHTML(untrusted_input);
+```
+
+If a sanitizing policy is installed as the default policy, this would
+auto-sanitize any HTML assignments.
+
+```js
+trustedTypes.createPolicy("default", { createHTML: new Sanitizer().bound() });
```

Comment:
Same issue, but the default policy is actually already called with three arguments, so the potential extension of `bound()` would be complex if there's existing code like that.